FabricExample/ios/FabricExample/Info.plist

```diff
@@ -38,7 +42,19 @@
 		</dict>
 	</dict>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string/>
+	<string></string>
```

Comment:
Why not `<string/>`?

Reply:
It's generated by Xcode or app template so idk